man/systemctl.xml

```diff
@@ -876,6 +876,18 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
             a similar filter for messages and might be more
             convenient.
             </para>
+            
+            <para>The Loaded: line in the output will show "loaded" if the unit has been loaded into memory and "not-found" if service can't be found.
```

Comment:
All values (loaded, not-found) should be wrapped in `<literal>` tags and drop the quotes.